manifest.json

```diff
@@ -6,7 +6,7 @@
       "dependencyManager": "cli-package",
       "appTemplate": "hello-world",
       "packageType": "Zip",
-      "useCaseName": "Serverless API"
+      "useCaseName": "Hello World Example"
```

Comment:
Should we make this `useCaseName` required for the new application templates?

Reply:
@jonife can you add this test into here?
https://github.com/aws/aws-sam-cli-app-templates/blob/master/tests/unit/test_json.py#L21